src/modules/common/actions/transferToken.ts

```diff
@@ -0,0 +1,96 @@
+import { DispatchRequest, getQuery, RequestAction } from '@redux-requests/core';
+import { createAction as createSmartAction } from 'redux-smart-actions';
+import { RootState } from 'store';
+import { Store } from 'redux';
+import { setAccount } from 'modules/account/store/actions/setAccount';
+import {
+  BounceErc721,
+  BounceErc1155,
+} from '../../web3/contracts';
+import { NftType } from 'modules/createNFT/actions/createNft';
+import { TransactionReceipt } from '@ethersproject/abstract-provider';
+
+export const transferToken = createSmartAction(
```

Comment:
The same as in the previous action.
`RequestAction<Data, TransformedData>`

Reply:
The same as before.